### PR TITLE
stub some hid service functions

### PR DESCRIPTION
**Pull request by emmauss**
for #55 , allows tetris to boot

### CODE REVIEW

File: Ryujinx.Core/OsHle/Services/Hid/ServiceHid.cs [hunk 1]
```diff
@@ -87,6 +90,40 @@ public long GetNpadJoyHoldType(ServiceCtx Context)
             return 0;
         }
 
+        public long SetNpadJoyAssignmentModeSingleByDefault(ServiceCtx Context)
+        {
+            long Unknown = Context.RequestData.ReadInt32();
```
Comment (gdkchan) [inline Ryujinx.Core/OsHle/Services/Hid/ServiceHid.cs]: Those values are not unknown, they are Id of type HidControllerID, see libnx implementation on hid.c. (more occurrences below).

Comment (emmauss) [reply Ryujinx.Core/OsHle/Services/Hid/ServiceHid.cs]: okay, i will check on it. I wonder why switchbrew was not updated.

Comment (emmauss) [reply Ryujinx.Core/OsHle/Services/Hid/ServiceHid.cs]: okay, i will check on it. I wonder why switchbrew was not updated.

**Comment by emmauss** [comment]
updated. I do not know what the unknowns in the final method are.

**Comment by gdkchan** [comment]
the controller id value is a 32 bits int, not a 64 bits long, also would be nice if you could use the HidControllerID type inside Ryujinx.Core.Hid (you just need to cast the int to HidControllerID, and use HidControllerID on the variable type too).

**Comment by gdkchan** [comment]
lgtm, thanks.

**Pull request merged**